### PR TITLE
Updating devDependencies

### PR DESCRIPTION
**Pull request by jason-fox**
Updating the `package.json` dev dependencies. No ESLint ... **yet**

```diff
   "devDependencies": {
-    "coveralls": "~3.0.3",
-    "istanbul": "~0.4.5",
-    "jshint": "~2.10.2",
-    "mocha": "6.1.4",
-    "nock": "10.0.6",
-    "should": "13.2.3",
-    "timekeeper": "2.2.0",
-    "watch": "~1.0.2",
-    "remark-cli": "~6.0.1",
-    "remark-preset-lint-recommended": "~3.0.3",
-    "textlint": "~11.3.1",
+    "coveralls": "~3.1.0",
+    "jshint": "~2.11.1",
+    "mocha": "8.0.1",
+    "nock": "13.0.3",
+    "nyc": "~15.1.0",
+    "remark-cli": "~8.0.1",
+    "remark-preset-lint-recommended": "~4.0.1",
+    "textlint": "~11.7.6",
     "textlint-filter-rule-comments": "~1.2.2",
     "textlint-rule-common-misspellings": "~1.0.1",
-    "textlint-rule-no-dead-link": "~4.6.1",
-    "textlint-rule-terminology": "~1.1.29",
+    "textlint-rule-no-dead-link": "~4.7.0",
+    "textlint-rule-terminology": "~2.1.4",
     "textlint-rule-write-good": "~1.6.2",
-    "sinon": "~9.0.2"
+    "should": "13.2.3",
+    "sinon": "~9.0.2",
+    "timekeeper": "2.2.0",
+    "watch": "~1.0.2"
   }
 }
```

### CODE REVIEW

**Comment by jason-fox** [comment]
Related: https://github.com/telefonicaid/iotagent-ul/pull/434

File: package.json [hunk 1]
```diff
@@ -59,22 +59,22 @@
...
-    "timekeeper": "2.2.0",
-    "watch": "~1.0.2",
-    "remark-cli": "~6.0.1",
-    "remark-preset-lint-recommended": "~3.0.3",
-    "textlint": "~11.3.1",
+    "coveralls": "~3.1.0",
+    "jshint": "~2.11.1",
+    "mocha": "8.0.1",
+    "nock": "13.0.3",
+    "nyc": "~15.1.0",
```
Comment (fgalan) [inline package.json:66]: Is this actually being used in scripts section?

File: CHANGES_NEXT_RELEASE [hunk 2]
```diff
@@ -1,3 +1,4 @@
+Overall update of update dev package dependencies
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:1]:
```suggestion
Overall update of dev package dependencies
```

File: package.json (hunk 1 above)
Comment (jason-fox) [reply package.json:66]: It is now - fixed be2edd3

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**